KalebIrwin-Power-signoff.md

```diff
@@ -0,0 +1,38 @@
+## Function of the subsystem 
+The power subsystem will provide the necessary energy for the components to operate. 
+
+## Constraints 
+1. Output three voltages 
+  12V to power the chosen microcontroller 
+  5V and 3V to power common sensors
+2. Runs continuously with minimal user interaction 
+3. Designed modularly
+  Operate with AC input 
+  Operate with a battery charged by a solar panel
+4. Provide ~485mW to microcontroller ~240mW + ~35mW for the communication and 210mW for three high current draw sensors simultaneously  
```

Comment:
Combine this with #1. A power value needs to be associated with each voltage value.